source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Application/Charges/Handlers/ChargePriceEventHandler.cs

```diff
@@ -113,9 +112,8 @@ public async Task HandleAsync(ChargePriceCommandReceivedEvent commandReceivedEve
                 operationsToBeConfirmed.Add(operation);
             }
 
-            await _chargePriceConfirmationService.SaveConfirmationsAsync(operationsToBeConfirmed).ConfigureAwait(false);
-            RaiseRejectedEvent(commandReceivedEvent, operationsToBeRejected, rejectionRules);
-            await _chargePriceNotificationService.SaveNotificationsAsync(operationsToBeConfirmed).ConfigureAwait(false);
+            HandleConfirmations(document, operationsToBeConfirmed);
+            HandleRejections(operationsToBeRejected, rejectionRules, document);
```

Comment:
Nit: my ocd would appreciate this 😄 

```suggestion
            HandleConfirmations(document, operationsToBeConfirmed);
            HandleRejections(document, operationsToBeRejected, rejectionRules);
```

Reply:
That's exactly what you would expect from an Inspector 😄